pkg/test/framework/components/echo/cmd/echogen/testdata/golden.yaml

```diff
@@ -162,4 +162,8 @@ spec:
           failureThreshold: 10
           periodSeconds: 1
           tcpSocket:
-            port: tcp-health-port
\ No newline at end of file
+            port: tcp-health-port
+      dnsConfig:
```

Comment:
as discussed offline, this is a ztunnel bug returning NOERROR instead of NXDOMAIN

Reply:
updated this PR to pull in https://github.com/istio/ztunnel/pull/603 now that it has merged, and removed the ndots hack